### PR TITLE
*: upgrade github.com/aws/aws-sdk-go-v2/config from v1.18 to v1.25

### PR DESCRIPTION
**Pull request by okJiang**
### What problem does this PR solve?

Issue Number: Ref #4399 

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test

### Release note

```release-note
None.
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -65,6 +65,8 @@ require (
 	gotest.tools/gotestsum v1.7.0
 )
 
+require github.com/aws/aws-sdk-go-v2/service/internal/accept-encoding v1.10.3 // indirect
```
Comment (rleungx) [inline go.mod:68]: need format

**Review by rleungx — APPROVED** [review]

**Review by bufferflies — APPROVED** [review]

**Comment by csuzhangxc** [comment]
@okJiang  Do we need to upgrade `github.com/aws/aws-sdk-go-v2/service/kms`? ref https://github.com/aws/aws-sdk-go-v2/issues/2370

**Pull request merged**